ci-scripts/dockerfiles/Lmod77_base.dockerfile

```diff
@@ -0,0 +1,39 @@
+#
+# LMod versions prior to 8.2 emitted Python commands differently, so we use this
+# Dockerfile to test the bindings of older versions
+#
+
+
+FROM ubuntu:20.04
+
+ENV TZ=Europe/Zurich
+ENV DEBIAN_FRONTEND=noninteractive
+ENV _LMOD_VER=7.7
+
+# ReFrame user
+RUN useradd -ms /bin/bash rfmuser
+
+# ReFrame requirements
+RUN \
+  apt-get -y update && \
+  apt-get -y install ca-certificates && \
+  update-ca-certificates && \
+  apt-get -y install gcc make git python3 python3-pip
+
+# Required utilities
+RUN apt-get -y install wget
+
+# Install Lmod
+RUN \
+  apt-get -y install lua5.3 lua-bit32:amd64 lua-posix:amd64 lua-posix-dev liblua5.3-0:amd64 liblua5.3-dev:amd64 tcl tcl-dev tcl8.6 tcl8.6-dev:amd64 libtcl8.6:amd64 lua-filesystem:amd64 lua-filesystem-dev:amd64 && \
+  wget -q https://github.com/TACC/Lmod/archive/${_LMOD_VER}.tar.gz -O lmod.tar.gz && \
+  tar xzf lmod.tar.gz && \
+  cd Lmod-${_LMOD_VER} && \
+  ./configure && make install && \
+  cd .. && rm -rf lmod.tar.gz Lmod-${_LMOD_VER} && \
+  apt-get clean && \
+  rm -rf /var/lib/apt/lists/*
+
+ENV BASH_ENV=/usr/local/lmod/lmod/init/profile
+
+USER rfmuser
```

Comment:
I would rather create the `rfmuser` in the reframe images and not those that install the different modules systems. Also I prefer hyphens over underscores in filenames, so can you rename all the base images to `-base`?

Reply:
This just changes the user from this step and beyond. Do you mean to move the actual user creation to the reframe images?

Reply:
> Do you mean to move the actual user creation to the reframe images?

Yes, I would say so, because otherwise you can't expand the image with things requiring elevated privileges, can you?

Reply:
No after changing the user you cannot install more things with `apt-get` for example.

Reply:
Yes, that's why I suggest moving this out from the base image.